app/controllers/users_controller.rb

```diff
@@ -13,19 +14,19 @@ def new
 
   def create
     @user = User.new(user_params)
-    @user.email.downcase!
+    @user.email.downcase! if @user.email
```

Comment:
just an FYI you can do safe navigation with Ruby instead of having if statements for these sorts of things. Here it would look like: `@user&.email&.downcase!`